### ISSUE TITLE
Is this moot for functions whose output already includes these extra params?

### ISSUE THREAD

**Issue by cottsak** [multiformats/multihash#72]
Like bcrypt for example where the output already includes the work factor:

```
$2a$10$...
```

**Comment by jbenet** [comment]
you could do it, just to be consistent. but you dont get much value if you're only storing bcrypt hashes

**Comment by cottsak** [comment]
@jbenet Could do what? Implement bcrypt in multihash?

**Comment by ericdmoore** [comment]
## TLDR:
I find lots of utility in the "to be consistent" argument.. If this is as simple as making a pull request against the multicodec table - please let me know, I will do so, and we can close this issue.

## Background 
If a project that is looking to adopt multihash as "standard for the project", which one might want to do if for no other reason than to have a system that accommodates different hash scheme - and can, at some later date migrate to a "more secure" hash - let alone for all the reasons described in the `readme.md`

To adopt this a the "standard way we hash" - it would be nice if `bcrypt` was available in the [multicodec table](https://github.com/multiformats/multicodec) perhaps where some standard bcrypt workfactors were coupled with the alogirthm - to add entries like `bcrypt11`, `bcrypt14` 

( NOTE: I just made those up - please take those as potentially terrible examples to use in practice - but a reasonable illustration of pairing a work factor and algorithm name.)
